docs/topics/modules/planning_development_work.adoc

```diff
@@ -68,36 +34,21 @@ image::gs_feature_eg.png[Example Feature]
 * <<creating_che_workspace-hello-world,*Modify the quickstart codebase*>>
 * <<reviewing_publishing_changes-hello-world,*Review and publish changes to your codebase*>>
 +
-Click the above tasks to see instructions on executing these tasks.
-+
 image::gs_tasks_egs.png[Example Tasks]
 
-. Similarly, for the *Fundamental* - *The application should be secure with appropriate licenses*, create a child experience*, feature, and task . The task could be titled <<analytics_stack_report,*Use analytics to identify and resolve security and license issues in the application*>>.
+. Similarly, select *Fundamental* in the *Add Work Item* pane, to create a fundamental work item with the title *The application should be secure with appropriate licenses* and use the in-line add option to create a child experience, feature, and task for it. The task can be titled <<analytics_stack_report,*Use analytics to identify and resolve security and license issues in the application*>>.
```

Comment:
Similarly is one of the words we should not use, unless it's clear what we're comparing it. In this case, I'd suggest not using it, since it's not really necessary anyway.

s/The task can be titled/Title the task or something else. We should avoid can/may/might/should/etc. and ensure the language is directive.

Reply:
* Unless we are actually talking about one of many options. In this case, I believe the usage is meant to be directive, i.e. do this rather than you can do this if you want to.